text/0030-protocol-messages.md

```diff
@@ -0,0 +1,243 @@
++ Feature name: `protocol-messages`
++ Start date: 2020-04-27
++ RFC PR: [iotaledger/bee-rfcs#30](https://github.com/iotaledger/bee-rfcs/pull/30)
++ Bee issue: [iotaledger/bee#70](https://github.com/iotaledger/bee/issues/70)
+
+# Summary
+
+This RFC introduces the IOTA protocol messages.
```

Comment:
A link to some official documentation or other implementation (IRI/Hornet) would be helpful here as a reader seeing this RFC before any other documentation might ask himself here: what exactly are **the** IOTA protocol messages?

Reply:
AFAIK there is no documentation, IRI doesn't really implement all of this protocol and there is no single place in HORNET where you can have an overview. Did you have something specific. in mind ? I'm not really sure what we could link here 🤔

Reply:
Unfortunately this is still a PR. Not sure if we want to link it in this status: https://github.com/iotaledger/protocol-rfcs/pull/1

Reply:
I'm basically introducing the same notions, IMO there is no point linking this one.

Reply:
If it's the only reference (and it seems properly written), and it doesn't hurt, then I would link it!!

Reply:
IMO it hurts. This RFC in the current state is very confusing. It is somehow IRI centered by speaking of a `network rewrite`. It doesn't add anything that is not said in this RFC. I am more than happy to update and link it later when it has been properly reviewed.

Reply:
From a quick glance there is even a big mistake in it...

Reply:
Okay, understand. Then I would suggest changing the sentence in a way such as if Bee introduced the IOTA protocol messages for the first time.

Reply:
But that's wrong ahah

Reply:
In a way, yes, but also in a way no. Bee is a new start anyway. In 10 years maybe no one even knows that there was a Java implementation before Bee.

Reply:
Any idea what to say then ?

Reply:
maybe something along the lines:
"This RFC introduces IOTA protocol messages that are described in detail further down below." ?